graze/ciff-renderer
# Render a functioning barcode. Currently assumed EAN13

Stolen code from the web repo to render a functioning barcode. 

Rendered images are to be sent to the vision system to be used as the base image it expects.

## QA
- `make composer-install`
- extract [these files](https://github.com/graze/ciff-renderer/files/811276/ciff-renderer-test.zip) in to a directory
- run  php -S localhost:8000 from that directory 
- browse to http://localhost:8000/test.php
- ensure the generated image has a scannable barcode. It should be prid 1588 with use by of today + 70 days (probably 110517)
